utils.js

```diff
@@ -47,8 +47,8 @@ function colorOptions({ color, succeedColor, failColor, spinnerColor }) {
 }
 
 function prefixOptions({ succeedPrefix, failPrefix }) {
-  succeedPrefix = succeedPrefix ? succeedPrefix : (process.platform === 'win32' ? '√' : '✓');
-  failPrefix = failPrefix ? failPrefix : (process.platform === 'win32' ?  '×' : '✖');
+  succeedPrefix = succeedPrefix ? succeedPrefix : (!terminalSupportsUnicode() ? '√' : '✓');
```

Comment:
Do you think we can refactor this function a bit? Having nested ternary operators can be a bit confusing to understand
We can do something like:
```js
if(terminalSupportUnicode()) {
  succeedPrefix = succeedPrefix || '✓';
  failPrefix = failPrefix || '✖';
} else {
  succeedPrefix = succeedPrefix || '√';
  failPrefix = failPrefix || '×';
}
```

I think it's easier to understand. It's not a blocker though :slightly_smiling_face:.